sklearn/ensemble/bagging.py

```diff
@@ -218,10 +219,14 @@ def __init__(self,
         self.bootstrap = bootstrap
         self.bootstrap_features = bootstrap_features
         self.oob_score = oob_score
+        self.warm_start = warm_start
         self.n_jobs = n_jobs
         self.random_state = random_state
         self.verbose = verbose
 
+        self.estimators_samples_ = []
+        self.estimators_features_ = []
```

Comment:
I added them to be similar to `self.estimators_` which is set in `BaseEnsemble.__init__`.

Where is a good place to set them instead? Main reason I added them is to make the reset-if-allocated-handling in `fit()` similar for `estimators_`, `estimators_samples_` and `estimators_features_`.